test/TensorFlowRuntime/tensor_api.swift

```diff
@@ -47,4 +44,4 @@ TensorNonTPUTests.testAllBackends("BroadcastTensor") {
   expectEqual(Tensor(repeating: 1, shape: [2, 3, 4]), target)
 }
 
-runAllTests()
+runAllTests()
```

Comment:
Could you revert changes to this file?

Reply:
Sorry I thought I had. It's reverted now. It was only missing some newlines so it shouldn't affect the tests.